charts/k8s-monitoring/templates/destinations/_destination_otlp.tpl

```diff
@@ -293,6 +305,25 @@ otelcol.exporter.loadbalancing {{ include "helper.alloy_name" .name | quote }} {
         }
       }
     }
+{{- end }}
+
+{{- if .processors.serviceGraphMetrics.enabled }}
+otelcol.exporter.loadbalancing {{ printf "%s_servicegraph" (include "helper.alloy_name" .name) | quote }} {
+  resolver {
+    kubernetes {
+      service = "{{ include "helper.k8s_name" (printf "%s-%s-servicegraph-alloy" $.Release.Name .name) }}"
```

Comment:
Should this match the full business that the tailSampler's `kubernetes { ... }` section gets?
That's essentially for handling:
* Names that are too long
* `nameOverride` and `fullnameOverride` settings